.gitignore

```diff
@@ -12,3 +12,4 @@ webpack-stats.json
 backup-*
 newrelic_agent.log
 .DS_Store
+db/
```

Comment:
that one is specific for you isn't it @jondot ? B/c you chose to place your DB right there, right?

Reply:
yup. figured it won't hurt. i usually keep these kind of assets very local